src/main/frontend/components/block.cljs

```diff
@@ -2298,9 +2298,7 @@
            (not advanced-query?)
            (string/blank? (:block/title block'))
            (seq (:block/title query)))
-      (text-block-title config
```

Comment:
I noticed that the file graphs no longer display the query builder. Not sure if it's related to this change or something else

Reply:
It should be fixed by https://github.com/logseq/logseq/commit/c854be62fc711f0ee40692fd738532bba0be0953.